CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master
 
+* Fixed issue where a released reference that was kept in `CLLocationManager` as `unowned` could cause crash. ([#1820](https://github.com/mapbox/mapbox-navigation-ios/pull/1820))
+* Replacing `performSelector:afterDelay:` mechanism that was [causing a crash](https://github.com/mapbox/mapbox-navigation-ios/issues/1662) with a DispatchTimerSource-based mechanism. ([#1820](https://github.com/mapbox/mapbox-navigation-ios/pull/1820))
```

Comment:
Similarly, this is a purely internal change. If we expect any reliability or stability improvements to result from this change, we can mention those improvements here; otherwise, the change isn’t actionable to the developer.

Reply:
```suggestion
* Replaces the `SimulatedLocationManager` timer mechanism with a `DispatchTimerSource`-based mechanism ([#1820](https://github.com/mapbox/mapbox-navigation-ios/pull/1820)), which fixes a crash that was caused by retaining the `SimulatedLocationManager` while it still had a dangling reference to its delegate (https://github.com/mapbox/mapbox-navigation-ios/issues/1662)
```

Reply:
I think the line above (as amended) adequately describes the developer impact of this PR. This PR’s description is the place to provide details about the crash’s root cause and the fix’s implementation.